### ISSUE TITLE
Release Nemotron 3 Super artifacts (models, dataset) on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [NVIDIA-NeMo/Nemotron#148]
Hi @NVIDIA-NeMo 🤗

Niels here from the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2604.12374.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models, datasets or demo for instance), you can also claim
the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

It'd be great to make all the Nemotron 3 Super checkpoints and the "Nemotron-Super-Post-Training-Data" dataset available on the 🤗 hub, to improve their discoverability/visibility.
We can add tags so that people find them when filtering https://huggingface.co/models and https://huggingface.co/datasets.

I found the `NVIDIA-Nemotron-3-Super-120B-A12B-BF16` and `Nemotron-3-Super-49B-v1` models, and the `Nemotron-Pretraining-Specialized-v1.1` dataset already on the Hub through your GitHub repository, which is fantastic!

However, the paper mentions additional Nemotron 3 Super 120B-A12B checkpoints in various quantization formats:
*   Nemotron 3 Super 120B-A12B NVFP4
*   Nemotron 3 Super 120B-A12B FP8
*   Nemotron 3 Super 120B-A12B Base BF16
And also a reward model:
*   Qwen3-Nemotron-235B-A22B-GenRM-2603

Could you please provide the Hugging Face links for these specific model variants and the reward model?

Additionally, the paper mentions the "Nemotron-Super-Post-Training-Data" dataset. I couldn't find a direct link for this specific dataset ID in the GitHub repository. Could you clarify if this dataset is available on Hugging Face and provide its URL, or if it's planned for release?

## Uploading models

See here for a guide: https://huggingface.co/docs/hub/models-uploading.

In this case, we could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverages the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

We encourage researchers to push each model checkpoint to a separate model repository, so that things like download stats also work. We can then also link the checkpoints to the paper page.

## Uploading dataset

Would be awesome to make the dataset available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

Let me know if you're interested/need any help regarding this!

Cheers,

Niels
ML Engineer @ HF 🤗

**Comment by shashank3959** [comment]
Hey @NielsRogge 

Thanks for reaching out!

Here are the links -

Checkpoints:
[Nemotron 3 Super 120B-A12B NVFP4](https://huggingface.co/nvidia/NVIDIA-Nemotron-3-Super-120B-A12B-NVFP4): post-trained and NVFP4 quantized model
[Nemotron 3 Super 120B-A12B FP8](https://huggingface.co/nvidia/NVIDIA-Nemotron-3-Super-120B-A12B-FP8): post-trained and FP8 quantized model
[Nemotron 3 Super 120B-A12B BF16](https://huggingface.co/nvidia/NVIDIA-Nemotron-3-Super-120B-A12B-BF16): post-trained model
[Nemotron 3 Super 120B-A12B Base BF16](https://huggingface.co/nvidia/NVIDIA-Nemotron-3-Super-120B-A12B-Base-BF16): base model
[Qwen3-Nemotron-235B-A22B-GenRM-2603](https://huggingface.co/nvidia/Qwen3-Nemotron-235B-A22B-GenRM-2603): GenRM used for RLHF


Data:
[Nemotron-Pretraining-Specialized-v1.1](https://huggingface.co/datasets/nvidia/Nemotron-Pretraining-Specialized-v1.1): a collection of synthetic datasets aimed to improve LLM capabilities in code concepts and algorithms, formal logic, economics, and multiple choice questions.
[Nemotron-Super-Post-Training-Data](https://huggingface.co/collections/nvidia/nemotron-post-training-v3): a collection of RL environments and SFT datasets targeting a broad range of agentic capabilities.

**Comment by NielsRogge** [comment]
Hey @shashank3959, thanks for providing the links.

I can confirm that most of the Nemotron 3 Super model checkpoints (BF16, FP8, NVFP4, Base BF16) are now linked to the paper page at https://huggingface.co/papers/2512.20856.

However, I noticed a couple of items that are not yet linked to the paper page:

1. The reward model [Qwen3-Nemotron-235B-A22B-GenRM-2603](https://huggingface.co/nvidia/Qwen3-Nemotron-235B-A22B-GenRM-2603) - could you add a reference to the paper (preferably https://huggingface.co/papers/2512.20856, or the arXiv abstract/PDF URL) in the model card README? This will make it automatically show up on the paper page.

2. No datasets are currently linked to the paper page. Would it be possible to add a reference to the paper in the README of the datasets in the [Nemotron-Post-Training-v3 collection](https://huggingface.co/collections/nvidia/nemotron-post-training-v3-6939b7b93382bac738eebd17)?

See https://huggingface.co/docs/hub/en/model-cards#linking-a-paper for more information on how to link artifacts to paper pages.

Thanks again for making these artifacts available on the Hub.
